commands/cli.go

```diff
@@ -48,7 +48,8 @@ func runHealthChecker(cliContext *cli.Context) error {
 	}
 
 	opts, err := parseOptions(cliContext)
-	if isSimpleError(err) {
+	if isDebugMode() {
+		opts.Logger.Infof("Note: To enable debug mode, set %s to \"true\"", ENV_VAR_NAME_DEBUG_MODE)
```

Comment:
I'm confused by this. Why do you return `err` right away in debug mode? And tell the user to enable debug mode?